app/services/audit_results.rb

```diff
@@ -39,7 +42,10 @@ class AuditResults
     UNEXPECTED_VERSION => "actual version (%{actual_version}) has unexpected relationship to %{addl} db version; ERROR!",
     INVALID_MOAB => "Invalid moab, validation errors: %{addl}",
     PC_PO_VERSION_MISMATCH => "PreservedCopy online moab version %{pc_version} does not match PreservedObject current_version %{po_version}",
-    ONLINE_MOAB_DOES_NOT_EXIST => "db has moab that is not found online"
+    ONLINE_MOAB_DOES_NOT_EXIST => "db has moab that is not found online",
+    CHECKSUM_MISMATCH => "The checksums for %{file1} does not match. Computed %{algorithm} checksum: %{computed_checksum}. Expected checksum: %{expected_checksum}",
+    FILE_NOT_ON_DISK => "%{xml_file} refers to a file (%{file1}) not found on disk",
+    FILE_NOT_IN_XML => "{file1} is not mentioned in %{xml_file}."
```

Comment:
constant name:    UNEXPECTED_FILE_IN_MOAB?   FILE_NOT_IN_MOAB_MANIFEST?   

"{file_path} is missing from %{manifest_file_path}."